app/upgrades.go

```diff
@@ -38,50 +30,15 @@ type appUpgrade struct {
 }
 
 var handlers = map[string]appUpgrade{
-	"ochre-rc1": { // upgrade for 1.13.0-rc3
-		Added: []string{group.ModuleName, rewardtypes.ModuleName, icacontrollertypes.StoreKey, icahosttypes.StoreKey},
-		Handler: func(ctx sdk.Context, app *App, plan upgradetypes.Plan) (module.VersionMap, error) {
-			versionMap := app.UpgradeKeeper.GetModuleVersionMap(ctx)
-			UpgradeICA(ctx, app, &versionMap)
-			ctx.Logger().Info("Starting migrations. This may take a significant amount of time to complete. Do not restart node.")
-			return app.mm.RunMigrations(ctx, app.configurator, versionMap)
-		},
-	},
-	"ochre-rc2": { // upgrade for 1.13.0-rc5
+	"quicksilver-rc1": { // upgrade for v1.15.0-rc1
```

Comment:
Super nitty on this...

1. There's a comment at the end of the `handlers` hash indicating that's where new handlers should go, and this one is being added to the top of the hash instead.
2. It'd be best to keep the `paua` and `paua-rc2` handlers in the same order as they currently are.

This will make this diff easier to process. `git` will correctly identify that the `ochre-rc1`, `ochre-rc2`, `ochre`, and `paua-rc1` upgrades are being deleted, nothing is changing with `paua-rc2` or `paua`, and `quicksilver-rc1` is being added with all new code.